Assets/Material/Characters/Character_Shadow.mat

```diff
@@ -9,18 +9,21 @@ Material:
   m_PrefabAsset: {fileID: 0}
```

Comment:
writing it here since I don't have anywhere else:
Shouldn't there be a script fix in this PR for a null check?

Reply:
sorry I see this is just a draft, I can review later

Reply:
Instead I removed those scripts from the GameObjects that generated the exceptions, since they did not need it